package.json

```diff
@@ -36,7 +36,8 @@
   "peerDependencies": {
     "@material-ui/core": "4.11.0",
     "@material-ui/icons": "4.9.1",
-    "react": "^16.0.0"
+    "react": "^16.13.1",
```

Comment:
Should this be `^`? I forget what's needed for peerDeps.

Reply:
I think that is okay to do this way. I checked material-ui and there's `^` in their peerDependencies https://github.com/mui-org/material-ui-pickers/blob/e875729b3492a4aedebec9f14589accc83475241/lib/package.json